packages/identity/tests/index.test.ts

```diff
@@ -24,20 +24,59 @@ describe("Identity", () => {
             expect(typeof identity.commitment).toBe("bigint")
         })
 
-        it("Should create deterministic identities from a secret hexadecimal (private key)", () => {
-            const identity = new Identity(privateKeyHexadecimal)
+        it("Should create a deterministic identity from a secret buffer (private key)", () => {
+            const identity = new Identity(privateKeyBuffer)
 
-            expect(typeof identity.privateKey).toBe("string")
-            expect(identity.privateKey).toHaveLength(64)
+            expect(typeof identity.privateKey).toBe("object")
+            expect(identity.privateKey).toHaveLength(6)
             expect(typeof identity.secretScalar).toBe("bigint")
             expect(identity.publicKey).toHaveLength(2)
             expect(typeof identity.commitment).toBe("bigint")
         })
 
+        it("Should create the same identity if the private key is the same", () => {
+            const identity = new Identity()
+
+            const identity2 = new Identity(identity.privateKey)
+
+            expect(identity.commitment).toBe(identity2.commitment)
+        })
+
         it("Should throw an error if the private key is not a string", () => {
             const fun = () => new Identity(32 as any)
 
-            expect(fun).toThrow("Parameter 'privateKey' is not a string, received type: number")
+            expect(fun).toThrow("Parameter 'privateKey' is none of the following types: Buffer, Uint8Array, string")
+        })
+    })
+
+    describe("# export", () => {
+        it("Should export an identity where the private key is a buffer", () => {
+            const identity = new Identity(privateKeyBuffer)
+
+            const privateKey = identity.export()
+
+            expect(typeof privateKey).toBe("string")
+            expect(Buffer.from(privateKey, "base64")).toStrictEqual(privateKeyBuffer)
+        })
+
+        it("Should export an identity where the private key is text", () => {
+            const identity = new Identity(privateKeyText)
+
+            const privateKey = identity.export()
+
+            expect(typeof privateKey).toBe("string")
+            expect(Buffer.from(privateKey, "base64")).toStrictEqual(privateKeyBuffer)
+        })
+    })
+
+    describe("# import", () => {
+        it("Should import an identity", () => {
```

Comment:
As I mentioned elsewhere, I'd suggest more "round-trip" tests for export + import to ensure you end up with the same identity2 regardless of whether the original identity was constructed using default (random) key, buffer key, or text key.